Лабораторная №5

PodbereznyjGA opened the pull request.

PodbereznyjGA committed:
Лабораторная №5

Selskij commented on Лабораторная 5/Task3.py:
```diff
@@ -0,0 +1,15 @@
+list_unique_numbers = get_unique_list_numbers()
+print(list_unique_numbers)
+print(len(list_unique_numbers) == len(set(list_unique_numbers)))
+
```
Видел решение буква в букву у ваших одногруппников

PodbereznyjGA replied:
Эти строки шли по умолчанию в задании лабораторной PyCharm. Я их поменяю, но какой смысл?

PodbereznyjGA replied:
Так же по аналогии с task5 создал аргументы функции

Selskij replied:
Дело не в принтах, а в теле функции, вплоть до названий переменных. Не воспринимайте на личный счёт, всем у кого решения очень похожи между собой я написал такое сообщение. Бал был снижен не за это задание а за task5.

PodbereznyjGA replied:
Хорошо, спасибо за обратную связь!

Selskij commented on Лабораторная 5/Task5.py:
```diff
@@ -0,0 +1,12 @@
+    pass_ = "".join(sample(words_, length_))
+    return pass_
+
+print(get_random_password())
```
Нужно реализовать передачу колличества знаков в пароле в виде аргумента. Чтобы можно было написать print(get_random_password(15)) и он бы выдал 15-и значный пароль

PodbereznyjGA replied:
Добавил аргумент length_

PodbereznyjGA committed:
Update Task3.py

Изменил согласно рекомендациям, так же по аналогии task5 сделал аргументы функции

PodbereznyjGA committed:
Update Task5.py

Сделал аргумент length_
